Src/GoogleApis.Auth/OAuth2/Requests/GoogleAuthorizationCodeRequestUrl.cs

```diff
@@ -49,6 +49,16 @@ public class GoogleAuthorizationCodeRequestUrl : AuthorizationCodeRequestUrl
         public string LoginHint { get; set; }
 
         /// <summary>
+        /// Gets or sets the include granted scopes to determine if this authorization request should use
+        /// incremental authorization (https://developers.google.com/+/web/api/rest/oauth#incremental-auth).
+        /// If true and the authorization request is granted, the authorization will include any previous 
+        /// authorizations granted to this user/application combination for other scopes. Default value is 

```

Comment:
I've removed it. I guess in my brain place I was trying to explain the resulting behavior of the URL and authorization, perhaps it would have been better as:

> Default *behavior* is false *which* will be omitted unless explicitly indicated.

But, that's a summary of the authorization behavior not the method, so I suppose it doesn't belong there.